spec/menu_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "menu.rb"
+
+describe Menu do
+  let(:dish) { double(:dish) }
+  let(:fish_dish) { double(:fish_dish) }
+  before(:each){
```

Comment:
This works perfectly well but there's a more idiomatic (and concise) way of achieving the same thing.